src/diagnostics/LogPointsManager.ts

```diff
@@ -104,8 +108,18 @@ class DebugSessionManager {
         };
     }
 
-    public async removeLogpoint(logpoint: ILogpoint): Promise<void> {
-        await this._debugSession.customRequest("removeLogpoint", logpoint.id);
+    public async removeLogpoint(logpoint: ILogpoint): Promise<boolean> {
+        const result: IRemoveLogpointResponse = await this._debugSession.customRequest("removeLogpoint", logpoint.id);
+        if (result.error !== undefined && result.error !== null) {
+            vscode.window.showErrorMessage("Cannot remove logpoint successfully. Please refer to DEBUG CONSOLE and https://aka.ms/logpoints for details.");
+            appserviceUtil.getOutputChannel().show();
+            const errorMessage = result.error.message === '' ?
```

Comment:
This is pretty similar to the addLogpoint logic. Would it make sense to put the error checking and UI logic on a higher layer? and leave the business logic here?

Reply:
The text message is different. Again, if we over-unify it, it might cause trouble for later translation. I prefer keep language template intact.